client/src/components/main/CreateNewPost.jsx

```diff
@@ -99,21 +98,18 @@ class CreateNewPost extends React.Component {
       lng: this.props.location.lng,
       profile_id: this.props.user.id,
       image_url: this.props.image_url,
-      landmark_id: landmark.id,
       storyID: this.storyID
     };
     console.log(post);
 
     console.log('(Client) Intiating POST Request! CREATING NEW POST');
 
-    this.setState({
-      postObject: {
-        post: post,
-        landmark: this.state.landmark
-      }
-    });
+    var postObject = {
```

Comment:
Should be refactored to a single state object. We can proceed with this for now.